src/chrome/content/rules/Andrew-Munsell.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset name="Andrew Munsell">
+
+	<target host="andrewmunsell.com" />
+	<target host="www.andrewmunsell.com" />
+
+	<rule from="^http://(?:www\.)?andrewmunsell\.com/"
+		to="https://www.andrewmunsell.com/" />
```

Comment:
https://www.andrewmunsell.com and https://andrewmunsell.com both work fine in testing. That, and the server 301s the non www version to www too